edc-extensions/control-plane-adapter-api/README.md

```diff
@@ -0,0 +1,20 @@
+# Control Plane Adapter API (EDR management)
+
+This module provide a new APIs on top of the EDC management APIs for dealing with EDRs token.
+
+The APIs are mounted in the same context of the `management` APIs. So no additional configuration is required.
```

Comment:
The extensions are added to the same context as the `management` APIs, so no additional configuration is required.